akka-distributed-cluster-docs/src/main/paradox/guide/1-event-sourced-shopping-cart.md

```diff
@@ -0,0 +1,194 @@
+# Part 1: Event Sourced Shopping Cart
+
+As the other features of Akka Distributed Cluster are build on top of Event Sourcing, let us start by implementing a shopping
+cart using the [Akka Event Sourced Behavior API](akka:typed/persistence.html). When this first step is complete, end users 
+will be able to add and remove items to a cart and finally check it out.
+
+We will build the cart as an Event Sourced entity, if you are unfamiliar with Event Sourcing, refer to the
+@extref[Event Sourcing section in the Akka guide](akka-guide:concepts/event-sourcing.html) for an explanation. 
+The [Event Sourcing with Akka video](https://akka.io/blog/news/2020/01/07/akka-event-sourcing-video) is also a good starting point for learning Event Sourcing.
+
+## Implementing an Event Sourced shopping cart
+
+### Commands
+
+Commands are the "external" API of an entity. Entity state can only be changed by commands. The results of commands are emitted as events. A command can request state changes, but different events might be generated depending on the current state of the entity. A command can also be validated and be rejected if it has invalid input or can’t be handled by current state of the entity.
```

Comment:
```suggestion
Commands are the "external" API of an entity. Entity state can only be changed by commands. The results of commands are emitted as events. A command can request state changes, but different events might be generated depending on the current state of the entity. A command can also be rejected if it has invalid input or can’t be handled by current state of the entity.
```

Simpler to just say it can be rejected. Validation is implicit.